README.md

```diff
@@ -114,6 +114,42 @@ If you want to customize the icons or the theme, refer to the [customization sec
 <script src="https://unpkg.com/buefy/dist/buefy.min.js"></script>
 ```
 
+
+### Installing Developer Snapshots from GitHub npm Registry to Access the Latest Features
+To incorporate the latest changes from the development branch of buefy-next, you can install developer snapshots from the GitHub npm registry. However, proceed with caution as these packages may be deleted or retracted without notice.
+
+Steps:
+1. Generate a Personal Access Token:
+    - Create a personal access token on GitHub with read access to the GitHub Packages registry.
+
+2. Configure npm:
+    - In your terminal, execute the following commands:
+```bash
+npm login --auth-type=legacy --registry=https://npm.pkg.github.com
+ USERNAME: <their GitHub username>
+ PASSWORD: <personal access token issued at Step 1>
+echo "@ntohq:registry=https://npm.pkg.github.com" > .npmrc
+```
+3. Select a Developer Release:
+    - Visit the Buefy-next package registry: https://github.com/ntohq/buefy-next/pkgs/npm/buefy-next on GitHub.
+Choose a stable developer release that suits your needs.
```

Comment:
There is no "stable" developer release. How about this?
> Choose any developer release that suits your needs.

And after that sentence, explain how the package names look like: `@ntohq/buefy-next@<package version>-<dev commit hash>`

`<package version>` is the intended release version that will include developer release versions starting with it; e.g., a developer release `@ntohq/buefy-next@0.1.3-commithash` will be included in the release `@ntohq/buefy-next@0.1.3`.

Reply:
> There is no "stable" developer release. How about this?
> 
> > Choose any developer release that suits your needs.
> 
> And after that sentence, explain how the package names look like: `@ntohq/buefy-next@<package version>-<dev commit hash>`
> 
> `<package version>` is the intended release version that will include developer release versions starting with it; e.g., a developer release `@ntohq/buefy-next@0.1.3-commithash` will be included in the release `@ntohq/buefy-next@0.1.3`.

That's fair. These changes sound good to me. I will make those changes when I get some free time